synapse/handlers/room_member.py

```diff
@@ -387,8 +404,11 @@ async def update_membership_locked(
                 raise SynapseError(403, "This room has been blocked on this server")
 
         if effective_membership_state == Membership.INVITE:
+            target_id = target.to_string()
+            self.ratelimit_invite(room_id, target_id)
```

Comment:
I wonder whether we should condition this on the `ratelimit` argument to this function. Unfortunately doing so would allow users to mass invite by supplying a long list of users to `/createRoom`. However not doing so might cause the server to get rate-limited accidentally?

Reply:
Oh good point. I can also limit the number of invites in `/createRoom` by the burst count.